nucliadb/src/nucliadb/search/search/find_merge.py

```diff
@@ -115,31 +115,13 @@ async def set_resource_metadata_value(
 
 class Orderer:
     def __init__(self):
-        self.boosted_items = []
-        self.items = []
+        self.items = {}
 
-    def add(self, key: Any):
-        self.items.append(key)
-
-    def add_boosted(self, key: Any):
-        self.boosted_items.append(key)
+    def add(self, key: Any, score: float):
+        self.items[key] = score
 
     def sorted_by_score(self) -> Iterator[Any]:
-        for key in sorted(self.items, key=lambda value: value[3], reverse=True):
-            yield key
-
-    def sorted_by_insertion(self) -> Iterator[Any]:
-        returned = set()
-        for key in self.boosted_items:
-            if key in returned:
-                continue
-            returned.add(key)
-            yield key
-
-        for key in self.items:
-            if key in returned:
-                continue
-            returned.add(key)
+        for key, _ in sorted(self.items.items(), key=lambda value: value[1], reverse=True):
```

Comment:
Maybe we can remove the `Orderer` class, as there's only one ordering algorithm, a function may be enough

Reply:
yup, agree!

Reply:
removed